chapter02/chapter02-00.adoc

```diff
@@ -0,0 +1,224 @@
+
+==== WildFly
+
+link:https://www.wildfly.org/[WildFly] is a flexible, lightweight, managed application runtime that offers full Jakarta EE and MicroProfile support. WildFly is designed for scalability and flexibility in both traditional and cloud-native environments. 
+
+link:https://www.wildfly.org/downloads/[WildFly Downloads] page offers the latest versions and documentation to get you started.
+
+==== Helidon
+
+Developed by Oracle, link:https://helidon.io/[Helidon] MP implements MicroProfile specifications. It provides a familiar programming model for Jakarta EE developers and enables efficient microservice development.
+
+link:https://helidon.io/docs/[Helidon Documentation] provides comprehensive resources to help developers get started with the framework, understand its core concepts, and develop microservices efficiently.
+
+==== Apache TomEE
+
+link:https://tomee.apache.org/[Apache TomEE] integrates several Apache projects with Apache Tomcat to provide a Jakarta EE environment. It offers support for MicroProfile, allowing developers to build and deploy microservices using the well-known Jakarta EE technologies with additional MicroProfile capabilities.
+
+link:https://tomee.apache.org/download.html[TomEE Downloads] and link:https://tomee.apache.org/microprofile-6.0/javadoc/[TomEE MicroProfile Documentation] page provide the necessary resources to get started with TomEE for MicroProfile development.
+
+=== MicroProfile Starter
+To kickstart your MicroProfile project, use the MicroProfile Starter to generate a sample project with your chosen server and specifications. This tool provides a customizable project structure and generates necessary boilerplate code and configuration.
+
+* Visit the link:https://start.microprofile.io/[MicroProfile] Starter page - the official website for generating the MicroProfile project templates.
+
+* Provide a `groupId` for your project, it's an identifier for your project and should be unique to avoid conflicts with other libraries or projects.
+
+TIP: Its recommended convention is to start your `groupId` with the reverse domain name of your organization (for example, `io.microprofile`).
+
+* Enter the 'artifactID', which is the name of your project (e.g., 'mp-ecomm-store').
+
+* Select the *Java SE version* your project will use.
+
+* Select the *MicroProfile version* you want to use. Ideally, you should choose the latest version for the most up-to-date features but also consider the runtime’s support.
+
+* Select the specifications you want to include in your project. These could be Config, Fault Tolerance, JWT Auth, Metrics, Health, Open API, Open Tracing, Rest Client. Choose what is relevant to your application.
+
+* Click the _Download_ button.
+
+* Unzip the generated project and import it into your IDE.
+
```

Comment:
Please also mention that you can use MicroProfile Starter via vscode extension [MicroProfile Extension Pack ](https://marketplace.visualstudio.com/items?itemName=MicroProfile-Community.vscode-microprofile-pack)or Intellij [MicroProfile plugin](https://plugins.jetbrains.com/plugin/13386-microprofile-starter)